ui/src/shared/components/graph_tips/GraphTips.tsx

```diff
@@ -1,24 +1,50 @@
-import React, {SFC} from 'react'
-import ReactTooltip from 'react-tooltip'
+// Libraries
+import React, {Component, createRef, RefObject} from 'react'
 
-const graphTipsText =
-  '<h1>Graph Tips:</h1><p><code>Click + Drag</code> Zoom in (X or Y)<br/><code>Shift + Click</code> Pan Graph Window<br/><code>Double Click</code> Reset Graph Window</p><h1>Static Legend Tips:</h1><p><code>Click</code>Focus on single Series<br/><code>Shift + Click</code> Show/Hide single Series</p>'
+// Components
+import {
+  Popover,
+  PopoverPosition,
+  PopoverInteraction,
+  PopoverType,
+} from '@influxdata/clockface'
 
-const GraphTips: SFC = () => (
-  <div
-    className="graph-tips"
-    data-for="graph-tips-tooltip"
-    data-tip={graphTipsText}
-  >
-    <span>?</span>
-    <ReactTooltip
-      id="graph-tips-tooltip"
-      effect="solid"
-      html={true}
-      place="bottom"
-      class="influx-tooltip"
-    />
-  </div>
-)
+interface Props {}
 
-export default GraphTips
+export default class GraphTips extends Component<Props> {
```

Comment:
Since this component doesn't have any state, we can simply keep it as a Pure or Functional component

Reply:
Thats good to know! Ill go ahead and change it :)

Reply:
Actually that change seems to break everything, so im not sure i can do it

Reply:
🤔 that's strange. Let's take this to slack